marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/searchNodes.sjs

```diff
@@ -206,11 +211,19 @@ result.map(item => {
   }
 })
 
-let finalQuery = cts.andQuery([ctsQuery, cts.tripleRangeQuery(null, sem.curieExpand("rdf:type"), entityTypeIRIs)]);
+//get total from base entities
+let resultBaseCounting = graphUtils.getEntityTypeIRIsCounting(entityTypeIRIs, ctsQuery);
+let resultBaseCountingVec = resultBaseCounting.toString().split(":");
```

Comment:
Suggestion: To avoid treating the result as a string maybe you can do ".toObject()" and then use the total as a property of your object to get the value.
I also saw something like this can work: fn.head(resultBaseCounting).total

Reply:
Done, thanks Aniela.